### ISSUE TITLE
Calculation of subtotal fails in mini-cart popup

### ISSUE THREAD

**Issue by jldmn** [magento/magento2#5106]
## Steps to reproduce
1. Add Product to cart
2. click on the shopping cart icon
## Expected result
1. correct subtotal shown at the top of the popup
## Actual result
1. no subtotal ist displayed
2. console logs:

```
knockout.js:3012 Uncaught ReferenceError: Unable to process binding "if: function (){return displaySubtotal() }"
Message: Unable to process binding "if: function (){return !display_cart_subtotal_excl_tax && display_subtotal_incl_tax }"
Message: display_subtotal_incl_tax is not defined
```

**Comment by Zaylril** [comment]
@jldmn This has already been reported see: https://github.com/magento/magento2/issues/3102 for solution

**Comment by sdzhepa** [comment]
Hello @jldmn
Seems this issue was fixed on 2.1 release also it works on current develop branch.

Probably there are some other steps or preconditions that were not described in the ticket.
Could You pls confirm is it resolved for You or not?
